soc/arm/Kconfig

```diff
@@ -41,6 +41,22 @@ config CPU_HAS_NRF_IDAU
 	  (IDAU: "Implementation-Defined Attribution Unit", in accordance with
 	  ARM terminology).
 
+if CPU_HAS_NRF_IDAU
+config NRF_SPU_FLASH_REGION_SIZE
+	hex
+	default 0x8000 if SOC_NRF9160
+	default 0x8000 if SOC_NRF5340_CPUAPP && NRF5340_CPUAPP_ERRATUM19
+	default 0x4000 if SOC_NRF5340_CPUAPP && !NRF5340_CPUAPP_ERRATUM19
```

Comment:
first `default` which match is used, therefore default `0x4000 if SOC_NRF5340_CPUAPP` would be enough

Reply:
done